### DISCUSSION TITLE
email-auth failed to send otp

### DISCUSSION THREAD

**Discussion by akramalmerdas** [saran-surya/email_auth_node#36] [Q&A]
Hello saran..
I am new to githup community and this my first discussion so please excuse me if made any mistake 
I am using your flutter package email auth and it was working fine using the test server but suddenly it went off before spending my 30 messages.
I thought it may be due to some error in the server so I decided to configure my own server and I followed your guide.
every thing seemed to work fine when I sat up the server and the results were as expected.
but when I tried to send an otp code from my flutter app I got the same message "email-auth >> failed to send otp".
I spent two days trying to figure out what went wrong and checking my Horuko log but it was all in vain.
then I conclude that it must be because I did not enable less secure access in my gmail because google has stopped this service.
I was hoping you may help me find another way to configure my own server or try fixing the test server please.

**Comment by saran-surya** [comment]
Hi,
Please be informed that the readme is the first place to check for all your
errors, also there was a patch made for test servers and production servers
will recieve an update soon.

Thanks again

**Reply by akramalmerdas** [reply]
the test server worked just fine again thank you..
still if you could fix the problem for the custom server soon it would be much appreciated.

**Comment by saran-surya** [comment]
I am currently occupied with other works, so it will surely take a while,
but you can expect it in coming weeks

**Reply by akramalmerdas** [reply]
it seems that the test server is down again.. still getting this message Failed to send OTP ❌..
please try at least to fix the test server...

**Comment by saran-surya** [comment]
Hi @akramalmerdas,
I am slightly busy with other works, and still finding time to fix them. Also the test servers had a hiccup, now the patch is made, and can you please try to check and update again. Thanks for you time

**Reply by akramalmerdas** [reply]
yeah it is working again but I am afraid of its rapid lagging..
Also, I want to thank you for your efforts in fixing the package constantly and for quick response to my messages even though you seem really busy...
I know I am a little annoying  but the thing is I have to present my graduation project after six days and it depends on your package for authentication, so please pardon me again if I panic a little when the server stops working ,but believe me there is a lot at stake here...
thanks again my friend . You are a true lifesaver...

**Comment by saran-surya** [comment]
Not to mention, also now almost everything is fixed and running. Also even
at the worst case, kindly contact me through LinkedIn or Instagram, I can
help you setup the production server, in case the test server fails. And
all the best for your presentation. ✌🏽

**Reply by akramalmerdas** [reply]
thanks a lot man. You have been a great help already... 
Still if there is a stable way to configure a production server already, please tell me how if it won't be a bother...

**Reply by saran-surya** [reply]
The steps are done, and also there is another input suggestion to directly use outlook, so if outlook has some better limits, we can try using them for the patch. or use gmail API that is free, and add outlook as a option to choose from during the configuration.. Will surely update soon. Thanks 🚀

**Reply by akramalmerdas** [reply]
My presentation was today and I am happy to tell you that I have aced it...
I just wanted to thank you again for your great help and more for your sincere care and quick responding despite being busy as you seem..
I really wish you all the best in your life bro and I hope our paths my cross again in the future with better circumstances...
Thanks again man.

**Reply by saran-surya** [reply]
Thanks again and All the best for your future ⭐
